e2e-tests/portal/cypress/integration/journeys/reports/unissued-facilities-reports.spec.js

```diff
@@ -21,7 +21,7 @@ context('Dashboard: Unissued facilities report', () => {
         name: 'abc-1-def',
         hasBeenIssued: false,
         value: 123,
-        currency: 'GBP',
+        currency: { id: 'GBP' },
```

Comment:
I believe this shouldn't have the `id` because this property is created in the controller

Reply:
I changed this part so currency is an object, was causing some issues